src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -145,4 +165,23 @@ void facetsLimit0() {
   void facetsLimit1() {
     facets(1);
   }
+
+  @Test
+  void delete() {
```

Comment:
Personally, I think it would be better (less ambiguous, more intent revealing, better describing) to be more specific that this test is to delete multiple users (using CQL) rather than a single user.

```suggestion
  void deleteMultipleUsersUsingCQL() {
```
```suggestion
  void multipleUsersCanBeDeletedUsingCQL() {
```